mimic/rest/identity_api.py

```diff
@@ -669,6 +688,521 @@ def list_tenants(self, request):
                 'message': ("No valid token provided. Please use the 'X-Auth-Token'"
                             " header with a valid token.")}})
 
+    @app.route('/v2.0/services', methods=['GET'])
+    def list_external_api_services(self, request):
+        """
+        List the available external services that endpoint templates
+        may be added to.
+
+        .. note:: Part of the OS-KSADM extension.
```

Comment:
When adding a note like this, it's helpful to include a hyperlink that provides a direct reference so that future maintainers don't need to google around for a term.